packages/website/versioned_docs/version-latest/introduction/style-guide.md

```diff
@@ -17,7 +17,7 @@ The guidelines in this article are used in TypeSpec Core libraries. You can use
 | scalar           | camelCase                                    | `scalar uuid extends string;`                    |
 | model            | PascalCase                                   | `model Pet {}`                                   |
 | model property   | camelCase                                    | `model Pet {furColor: string}`                   |
-| enum             | PascalCase                                   | `model Pet {furColor: string}`                   |
+| enum             | PascalCase                                   | `enum Direction {}`                              |
 | enum member      | camelCase                                    | `enum Direction {up, down}`                      |
```

Comment:
Although all examples mentioned here https://microsoft.github.io/typespec/language-basics/enums has PascalCase members :confused:

Reply:
Good catch, sounds like we need to update the other docs too. They were written before we came to an agreement for the style guide